docs/cloud/runtime-troubleshooting-with-functions.md

```diff
@@ -32,10 +32,12 @@ The following is required to be able to run Functions from Netdata Cloud.
 2. You are directed to the **Functions** tab
 3. Follow the above instructions from step 3.
 
+> ⚠️ If you get an error saying that your node can't execute Functions please check the [prerequisites](#prerequisites).
+
 ## Related Topics
 
 ### **Related Concepts**
-- [Netdata Functions](https://github.com/netdata/netdata/blob/master/docs/concepts/guided-troubleshooting/netdata-functions.md)
+- [Netdata Functions](https://github.com/netdata/netdata/blob/master/docs/cloud/netdata-functions.md)
 
 #### Related References documentation
-- [External plugins overview](https://github.com/netdata/netdata/tree/master/collectors/plugins.d#function)
+- [External plugins overview](https://github.com/netdata/netdata/blob/master/docs/nightly/references/collectors-references/plugins.d/#function)
```

Comment:
This link is 404

Reply:
seems @tkatsoulas made a suggestion will try that one

Reply:
fixed by putting the direction to the README.md file